src/util/schemas/ChannelModifySchema.ts

```diff
@@ -25,4 +25,7 @@ export interface ChannelModifySchema {
 	default_auto_archive_duration?: number;
 	flags?: number;
 	default_thread_rate_limit_per_user?: number;
+	auto_archive_duration?: 60 | 1440 | 4320 | 10080;
```

Comment:
```suggestion
	auto_archive_duration?: number;
```
no need to carry over 1h/1d/3d/1w restriction to fosscord